src/app/shared/pipes/time-duration/timestamp.ts

```diff
@@ -25,6 +25,9 @@ export class Timestamp {
   }
 
   parseToSeconds(value: ParseTimestampResult): number {
```

Comment:
@dawidgil this was sending 0 to ICL, so ICL was storing 0 as default validity period and then SB was then using it as a timestamp. If everything is 0 here, it sets the timestamp to null.

Reply:
Yeah, i didn't think about this case. Thanks!